docs/getting-started-guides/ubuntu/index.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: Kubernetes on Ubuntu
+---
+
+{% capture overview %}
+There are multiple ways to run a Kubernetes with Ubuntu. These pages explain how to deploy Kubernetes on Ubuntu on multiple public and private clouds, as well as bare metal. 
```

Comment:
I'm putting together a proposal that I hope to get out to sig-docs within a week.